projects/packages/connection/src/class-plugin-storage.php

```diff
@@ -7,6 +7,7 @@
 
 namespace Automattic\Jetpack\Connection;
 
+use Jetpack_options;
```

Comment:
Doesn't seem to matter, but still just in case
```suggestion
use Jetpack_Options;
```